Get-NetView.psm1

```diff
@@ -230,62 +234,55 @@ function Start-Thread {
+        $null = $ps.AddScript("Set-Location $(Get-Location)")
+        $null = $ps.AddScript($ExecFunctions) # import into thread context
+        $null = $ps.AddScript($ScriptBlock, $true).AddParameters($Params)
 
-    $async = $ps.BeginInvoke()
+        $async = $ps.BeginInvoke()
 
-    return @{Name=$ScriptBlock.Ast.Name; AsyncResult=$async; PowerShell=$ps}
+        return @{Name=$ScriptBlock.Ast.Name; AsyncResult=$async; PowerShell=$ps}
+    }
 } # Start-Thread()
 
 function Show-Threads {
     [CmdletBinding()]
     Param(
-        [parameter(Mandatory=$true)] [Hashtable[]] $Threads,
-        [parameter(Mandatory=$false)] [Switch] $Sequential
+        [parameter(Mandatory=$false)] [Hashtable[]] $Threads
     )
 
-    if ($Sequential) {
-        $Threads | foreach {
-            $_.Powershell.Streams.Error | Out-Host # blocks until thread completion
-            $_.Powershell.Streams.Warning | Out-Host
-            $_.Powershell.Streams.Information | Out-Host
-            $_.PowerShell.Streams.ClearStreams()
-            $_.PowerShell.EndInvoke($_.AsyncResult)
-        }
-    } else {
-        $mThreads = [Collections.ArrayList]$Threads
-
-        while ($mThreads.Count -gt 0) {
-            for ($i = 0; $i -lt $mThreads.Count; $i++) {
-                $thread = $mThreads[$i]
-
-                $thread.Powershell.Streams.Warning | Out-Host
-                $thread.Powershell.Streams.Warning.Clear()
-                $thread.Powershell.Streams.Information | foreach {Write-CmdLog "$_"}
-                $thread.Powershell.Streams.Information.Clear()
-
-                if ($thread.AsyncResult.IsCompleted)
-                {
-                    # Accessing Streams.Error blocks until thread is completed
-                    $thread.Powershell.Streams.Error | Out-Host
-                    $thread.Powershell.Streams.Error.Clear()
-
-                    $thread.PowerShell.EndInvoke($thread.AsyncResult)
-                    $mThreads.RemoveAt($i)
-                    $i--
-                }
+    $mThreads = [Collections.ArrayList]$Threads
+
+    while ($mThreads.Count -gt 0) {
+        for ($i = 0; $i -lt $mThreads.Count; $i++) {
+            $thread = $mThreads[$i]
+
+            $thread.Powershell.Streams.Warning | Out-Host
+            $thread.Powershell.Streams.Warning.Clear()
+            $thread.Powershell.Streams.Information | foreach {Write-CmdLog "$_"}
+            $thread.Powershell.Streams.Information.Clear()
+
+            if ($thread.AsyncResult.IsCompleted)
+            {
+                # Accessing Streams.Error blocks until thread is completed
+                $thread.Powershell.Streams.Error | Out-Host
+                $thread.Powershell.Streams.Error.Clear()
+
+                $thread.PowerShell.EndInvoke($thread.AsyncResult)
+                $mThreads.RemoveAt($i)
+                $i--
             }
-            Start-Sleep -Milliseconds 15
         }
+        Start-Sleep -Milliseconds 33
```

Comment:
How was 33ms determined?  Does this need to be a tunable?  At minimum, please add a comment for how the value was chosen.

Reply:
1000 / 33 = ~30 times per second. The value is purely cosmetic: if it's too low, console output may appear "choppy" or delayed.

Reply:
Please add a comment in the code to capture this.  Else it's a magic value.  Better yet, make it a descriptive named variable for readability.